### DISCUSSION TITLE
WT32-S1 problem to flash with compield version with neopool

### DISCUSSION THREAD

**Discussion by ollyins** [arendst/Tasmota#18249] [Q&A]
Hi, i tried to flash the WT32-ETH01 V1.2.. when doing this via the Tasmota web installer.. i tried tasmota english... its workig.. but when i activte the template later to get the eth port to configure.. its not working! i also need the neopool addon.. so i compiled with gitpod a complete version.... with tasmota32 and the following 
user_config_override.h

```
// -- Neopool Addon --------------------
#ifndef USE_NEOPOOL
#define USE_NEOPOOL                       // Add support for Sugar Valley NeoPool Controller (+6k flash, +60 mem)
#endif

// -- Tasmota with Ethernet on WT32-ETH01 --------------------
#define USE_ETHERNET
  // Wireless-Tag WT32-ETH01
  #define ETH_TYPE          0                    // [EthType] 0 = ETH_PHY_LAN8720, 1 = ETH_PHY_TLK110, 2 = ETH_PHY_IP101
  #define ETH_ADDRESS       1                    // [EthAddress] 0 = PHY0 .. 31 = PHY31
  #define ETH_CLKMODE       0                    // [EthClockMode] 0 = ETH_CLOCK_GPIO0_IN, 1 = ETH_CLOCK_GPIO0_OUT, 2 = ETH_CLOCK_GPIO16_OUT, 3 = ETH_CLOCK_GPIO17_OUT

``` 
But when i ried to flash the WT32 from his webinterface with this new binary i get "upload failed" and the device is in safeboot.
any ideas how to do? What i did wrong maybe?

**Comment by sfromis** [comment]
And the binary you're trying to upgrade to is named specifically `tasmota32.bin`?

**Reply by ollyins** [reply]
it has another name.. i changed to identify the bin later... but now it was working... i have the tasmota runing, also with neopool addon and wifi.. i put also the script to turn off wifi at start if lan is available but i didnt checked it without wifi... tomorrow i will substitute my esp32 in the garage with this baby so that i can turn off there the provisional accesspoint... lets see what its doing... i only have to find a good power supply . atm the esp32 is powered via usb cable... so i will look for a step down to 5 V or 3.3... dont kow what i did there for the ttl modbus converter.... have to check... thanks
